qannotate/src/au/edu/qimr/qannotate/modes/Vcf2maf.java

```diff
@@ -145,37 +160,37 @@ public Vcf2maf( Options option) throws Exception {
 			
 			for (final VcfRecord vcf : reader) {
 				
-	    			noIn ++;
-	    			SnpEffMafRecord maf = converter(vcf);
-	    			String sMaf = maf.getMafLine();
-	    			out.println(sMaf);
-	    			noOut ++;
-	    			int rank = Integer.parseInt(maf.getColumnValue( MafElement.Consequence_rank));//40
-	    			boolean isConsequence = isConsequence(maf.getColumnValue(MafElement.Transcript_BioType), rank); //55
-	    			boolean isSomatic = maf.getColumnValue(MafElement.Mutation_Status).equalsIgnoreCase(VcfHeaderUtils.INFO_SOMATIC); //26
-	    			if (isHighConfidence(maf)) {
-	    				if (isSomatic){
-	    					out_SP.println(sMaf);
-	    					outSPVcf.add(vcf);
-	    					no_SHC ++;
-	    					
-	    					if(isConsequence){
-	    						out_SPC.println(sMaf);
-	    						outSPCVcf.add(vcf);
-	    						no_SHCC ++;
-	    					}
-	    				} else {
-	    					out_GP.println(sMaf);
-	    					outGPVcf.add(vcf);
-	    					no_GHC ++; 
-	    					 
-	    					if(isConsequence){
-	    						out_GPC.println(sMaf);
-	    						outGPCVcf.add(vcf);
-	    						no_GHCC ++;
-	    					}
-	    				}   
-	    			} 
+    			noIn ++;
+    			SnpEffMafRecord maf = converter(vcf);
+    			String sMaf = maf.getMafLine();
+    			out.println(sMaf);
+    			noOut ++;
+    			int rank = Integer.parseInt(maf.getColumnValue( MafElement.Consequence_rank));//40
+    			boolean isConsequence = isConsequence(maf.getColumnValue(MafElement.Transcript_BioType), rank); //55
+    			boolean isSomatic = maf.getColumnValue(MafElement.Mutation_Status).equalsIgnoreCase(VcfHeaderUtils.INFO_SOMATIC); //26
+    			if (isHighConfidence(maf)) {
+    				if (isSomatic){
+    					out_SP.println(sMaf);
+    					outSPVcf.add(vcf);
+    					no_SHC ++;
+    					
+    					if(isConsequence){
+    						out_SPC.println(sMaf);
+    						outSPCVcf.add(vcf);
+    						no_SHCC ++;
+    					}
+    				} else {
+    					out_GP.println(sMaf);
+    					outGPVcf.add(vcf);
+    					no_GHC ++; 
+    					 
+    					if(isConsequence){
+    						out_GPC.println(sMaf);
+    						outGPCVcf.add(vcf);
+    						no_GHCC ++;
+    					}
+    				}   
+    			} 
```

Comment:
accidental deindent?

Reply:
looks ok in my IDE - does it look dodgy in github or in an IDE?

Reply:
I think it's because ll175-205 (i.e. the body of the for loop) have a leading 4 spaces instead of tabs. Now that I look there's a few other spaces vs tabs inconsistencies in the file though so I'm not too fussed about this one; just leapt out in the diff because in my IDE that makes the closing brace on line 205 visually line up with that below on 206